decidim-debates/app/controllers/decidim/debates/debates_controller.rb

```diff
@@ -123,7 +123,7 @@ def default_search_params
       def default_filter_params
         {
           search_text: "",
-          origin: %w(official citizens user_group),
+          origin: %w(official participants user_group),
```

Comment:
Maybe you should change or remove also the `decidim.debates.debates.filters.citizens` keys in the `decidim-debates/config/locales`

Reply:
Already done here: 

![image](https://user-images.githubusercontent.com/717367/149143944-cab782a1-594a-4889-9c3d-01681697a1b5.png)

Reply:
The keys are also present in the other I18n  files ...

Reply:
Closing as per : https://github.com/decidim/decidim/pull/8697#discussion_r783009123